app/models/team.rb

```diff
@@ -51,6 +52,10 @@ def has_member?(user)
     members.include?(user)
   end
 
+  def is_qualified_to_receive_notifications?
+    !self.created_by_event?
```

Comment:
Any reason for not using `self.created_by_user?` here? To my mind, this would increase the readability by avoiding the negation.

Reply:
I added created_by_user later... But you're definitely right, I'll change it!

Reply:
Well, I just recognized that `created_by_user` is a scope for `Team` and no instance variable. So `self.created_by_user?` will result in lots of errors. I will revert to the original `!self.created_by_event?` or do you have any other ideas?

Reply:
I would agree with @lheilmann here. The only way to use created_by_user would be to intersect with the scope... not very useful.

Reply:
Aah, okay – sorry, I didn't notice that `created_by_user` is only a scope and not an instance variable. Then it's fine to leave it at the original `!self.created_by_event?`. 👍

Reply:
if this is purely about readability, what you could do is use `not` instead of `!` for your negation here. the operators are slightly different in precedence, but should both work here...

Reply:
Good idea! :+1: